restapi/api.go

```diff
@@ -63,6 +63,7 @@ func Router(db *gorm.DB, locationFinder location.LocationFinder) (http.Handler,
 		rest.GetHandler(r, "/story/{storyID}", api.ReturnStoryByID)
 		rest.DeleteHandler(r, "/story/{storyID}", api.DeleteStoryByID)
 		rest.PutHandler(r, "/story/{storyID}", api.EditStoryByID)
+		rest.PutHandler(r, "/publish", api.PublishStories)
```

Comment:
Is the endpoint name "/publish" fine or should it be smth else?

Reply:
maybe `/stories/publish` would be better